src/Car.java

```diff
@@ -1,14 +1,14 @@
 
 public class Car {
-	
+
 	private int id;
 	private String marka;
```

Comment:
It's better to use English words against transliterate words: marka = model, and you can change your second "model" to type.